menas/ui/service/GenericService.js

```diff
@@ -33,9 +33,40 @@ var GenericService = new function () {
     })
   };
 
+  this.runStatusFormatter = function(sStatus) {
```

Comment:
JavaScript does have a switch statement, that could be better here and is supposedly faster

Reply:
Which also bloats the code significantly (as much as I'd prefer to use a construct closer to pattern matching).

Reply:
I think there is less tbh. I know it is 2 lines longer, but in general less useless text.
```javascript
switch(sStatus) {
    case "failed":                  return "Failed";
    case "successful":              return "Successfull";
    case "successfulWithErrors":    return "Sucessful with errors";
    case "running":                 return "Running";
    case "stdSuccessful":           return "Standardization successful";
    default:                        return sStatus;
}
```

And you don't have to tab it if you don't want to.

Reply:
Or check the approach in this [totaly unsuspicious link](https://ultimatecourses.com/blog/deprecating-the-switch-statement-for-object-literals#Object_Literal_lookups)